src/content/plugins/limit-chunk-count-plugin.md

```diff
@@ -43,23 +44,7 @@ module.exports = {
 
 ### `minChunkSize`
 
-`number`
-
-Set a minimum chunk size in bytes.
-
-__webpack.config.js__
-
-```javascript
-const webpack = require('webpack');
-module.exports = {
-  // ...
-  plugins: [
-    new webpack.optimize.LimitChunkCountPlugin({
-      minChunkSize: 1000
-    })
-  ]
-};
-```
+To keep chunk size above the specified limit by merging chunks that are smaller than the limit, use [MinChunkSizePlugin)[/plugins/min-chunk-size-plugin]
```

Comment:
Why is this needed here if its not done via this plugin anymore?

Reply:
Aren’t plugins a valid way of configuring Webpack anymore?

Reply:
i mean if there is a plugin that does exactly this, move this info to its page?

Reply:
There is already such info on the page I linked to. I've added the link to make migration from old API to new one a bit easier. It's more like "hey, this feature was moved somewhere else, if you search for it, look at here".

Reply:
Okay, that is still what i think is not really obvious from the text. The text says: "To do X, use plugin X", when you are saying it is intended to be "Hey, feature X has moved to plugin X". Or am i reading this wrong?

Reply:
You're reading it right. I'll try to make the message clearer :)